src/pages/core/guides/oracle-plugin-example.md

```diff
@@ -0,0 +1,375 @@
+                &[signer_seeds]
+            ),
+            REWARD_IN_LAMPORTS
+        )?
+    }
+
+    Ok(())
+}
+```
+
+This instruction functions similarly to the create_oracle instruction but with added checks. Based on the response from the is_us_market_open function, it verifies if the state was already updated. If not, it updates the state.
+
+The second part of the instruction checks if is_within_15_minutes_of_market_open_or_close is true and if there are enough lamports in the reward vault to pay the cranker. If both conditions are met, it transfers the reward to the cranker; otherwise, it does nothing.
+
+### Create the NFT
+
+Last part of this joruney will be to create a collection and point it to the Oracle account so every asset we include in that collection will follow the custom Oracle rule!
+
+Let's start by setting up your environment to use Umi. (Umi is a modular framework for building and using JavaScript clients for Solana programs. Learn more [here](../../umi/getting-started))
+
+```ts
+import { createSignerFromKeypair, signerIdentity } from '@metaplex-foundation/umi'
+import { createUmi } from '@metaplex-foundation/umi-bundle-defaults'
+
+// SecretKey for the wallet you're going to use 
+import wallet from "../wallet.json";
+
+const umi = createUmi("https://api.devnet.solana.com", "finalized")
+
+let keyair = umi.eddsa.createKeypairFromSecretKey(new Uint8Array(wallet));
+const myKeypairSigner = createSignerFromKeypair(umi, keyair);
+umi.use(signerIdentity(myKeypairSigner));
+```
+
+Next, we create the collection including the Oracle Plugin using the `CreateCollection` instruction:
+
+```ts
+// Generate the Collection PublicKey
+const collection = generateSigner(umi)
+console.log("Collection Address: \n", collection.publicKey.toString())
+
+const oracleAccount = publicKey("...")
+
+// Generate the collection
+const collectinTx = await createCollection(umi, {
+    collection: collection,
+    name: 'My Collection',
+    uri: 'https://example.com/my-collection.json',
+    plugins: [
+        {
+            type: "Oracle",
+            resultsOffset: {
+                type: 'Anchor',
+            },
+            baseAddress: oracleAccount,
+            authority: {
+                type: 'UpdateAuthority',
+            },
+            lifecycleChecks: {
+                transfer: [CheckResult.CAN_REJECT],
+            },
+            baseAddressConfig: undefined,
+        }
+    ]
+}).sendAndConfirm(umi)
+
+// Deserialize the Signature from the Transaction
+let signauture = base58.deserialize(collectinTx.signature)[0];
+console.log(signauture);
+```
+
+## Conclusion
+
+Congratulations! You are now equipped to create an NFT collection that trades only during US market hours using the Oracle Plugin. If you want to learn more about Core and Metaplex, check out the developer portal [here](/core/getting-started).
+
+Follow my journey and stay updated with my next tutorial on [Twitter](https://twitter.com/L0STE_)!
```

Comment:
Not sure if we should add our personals here since it is the official doc page.

Reply:
Right this make sense, i default to putting it but it doesn't make sense